main_view.html

```diff
@@ -8,44 +8,324 @@
     <link rel="stylesheet" href="style_main_view.css">
 </head>
 <body>
-
+<div class="welcome_view">
     
 <header>
-
 <div class="menu">
   <a href="#" class="logo"> Welcome </a>  
   <nav class="menu_img">
   <img src="images/Enabledmenu_bar.svg" alt="">  
   </nav>
     </div>
-
 </header>
 
 <main class="background_main">
-
-
-<section>
+<div>
     <h1 class="introduction"> Hey there. I’m Jose Zepeda <span>I’m a software developer</span> </h1>
     <p class="content_introduction">  
     I can help you build a product , feature or website 
     Look through some of my work and experience! If you like 
     what you see and have a project you need coded, 
     don’t hestiate to contact me.</p>
-
 <ul class="social_media_icons">
 <li class="twitter"><a href="#">  <img src="images/twitter.svg" alt="twitter"></a> </li>    
 <li class="linkedin"><a href="#"><img src="images/Linkedin.svg" alt="linkedin"></a></li>    
 <li class="medium"><a href="#"><img src="images/medium.svg" alt="medium"></a></li> 
 <li class="github"><a href="#"><img src="images/github.svg" alt="github"></a></li>    
 <li class="angellist"><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li> 
 </ul>
-
 <a class="row_down" href="#"><img  src="images/Action.svg" alt="Action" ></a> 
-
-</section>
+</div>
 </main>
-
+</div>
  
+
+<div class="recent_works">
+<div class="recent_works_elements">
+
+  <h1 class="title_header">My Recent Works</h1> 
```

Comment:
- [ ] Please use `h2` tag instead of `h1` to follow the HTML [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1: